src/Accounts/Accounts/Default/GetAzureRmDefault.cs

```diff
@@ -35,9 +37,21 @@ public class GetAzureRMDefaultCommand : AzureRMCmdlet
         [Parameter(ParameterSetName = ResourceGroupParameterSet, Mandatory = false, HelpMessage = "Display Default Resource Group", ValueFromPipelineByPropertyName = true)]
         public SwitchParameter ResourceGroup { get; set; }
 
+        protected override IAzureContext DefaultContext
```

Comment:
Is is possible to add tests for these?  It would be nice to not regress on these in the future.